### ISSUE TITLE
There is no need to repeat "Tool" for each tool in the Toolbox menu

### ISSUE THREAD

**Issue by FirasH** [synfig/synfig#1151]
Currently the Toolbox menu contains all the tools where the entries are labeled as follows:

"Name1 Tool"
"Name2 Tool"
...
"NameN Tool"

I think it would be cleaner to have something like:
"Name1"
"Name2"
...
"NameN"

**Comment by FirasH** [comment]
I can easily create a pull request for this. Does it make sense to you developers?

**Comment by ice0** [comment]
Just checked GIMP and Photoshop - both use the %s Tool for a tooltip, so I suggest to leave them as is.

**Comment by FirasH** [comment]
I agree with you about the tooltip to keep "%s Tool" in toolbox on the left.

I was refering to the menu list "Toolbox" from the toolbar as having a long list of "%s Tool" one on top of each other "heavy" to look at.

![1151](https://user-images.githubusercontent.com/6705690/74668351-56723380-51a5-11ea-9b22-6938d96f7f45.png)

**Comment by morevnaproject** [comment]
> I was referring to the menu list "Toolbox" from the toolbar as having a long list of "%s Tool" one on top of each other is "heavy" to look at.

Agree. 

Actually, I have some thoughts about Synfig's menu structure: we ahve a lot of entries that aren't used by people. For example, why have "Navigation" menu, while users prefer to click on buttons right below the workarea? Why have Toolbox menu, while we have Toolbox Panel? Why have "Layer" - "Cut", while it is available via right-click and also in "Edit" - "Cut"?  I think Synfig menu can be reduced. This will unclutter the interface and make application easier to learn.

**Comment by FirasH** [comment]
@morevnaproject I agree with you about the menus, I have never used the Navigation menu for example...

Is there a place where we can keep a note about those ideas so we have everything sorted for the menu update/redesign?

**Comment by ice0** [comment]
Maybe add new Issue and set Roadmap for 1.6 for it? At least we do not release until it will be resolved.

**Comment by neesaaa** [comment]
if this still an issue I can take it . plus I think changing the name in the drop-down menu would change it in the toolbox-text that appears on mouse-hover

**Comment by mohamedAdhamc** [comment]
> if this still an issue I can take it . plus I think changing the name in the drop-down menu would change it in the toolbox-text that appears on mouse-hover @mohamedAdhamc @ice0

I believe you should be able to change the text in the menu and still keep the old tool-tip for the Toolbox.
